PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/addendum/AdViewUtils.java

```diff
@@ -227,6 +230,53 @@ public void onReceiveValue(@Nullable String html) {
         });
     }
 
+    /**
+     * {@link PrebidNativeAdListener} will be called only once.
+     * {@link PrebidNativeAdListener#onPrebidNativeLoaded(PrebidNativeAd)} when Cached Ad is found and is valid
+     * {@link PrebidNativeAdListener#onPrebidNativeNotValid()} (PrebidNativeAd)} when Cached Ad is found and but is invalid
+     * and {@link PrebidNativeAdListener#onPrebidNativeNotFound()} ()} (PrebidNativeAd)} when Cached Ad is not found
+     */
+    public static void iterateWebViewListAsync(final List<WebView> webViewList, final int index, final PrebidNativeAdListener listener) {
+
+        final WebView webView = webViewList.get(index);
+
+        webView.evaluateJavascript(INNER_HTML_SCRIPT, new ValueCallback<String>() {
```

Comment:
`Call requires API level 19 (current min is 16)`

Reply:
Added a check for `Build.VERSION_CODES.KITKAT`. If its lower than KITKAT version, then we will be returning `onPrebidNativeNotFound()`

Reply:
I think that error should be more informative. 
For example I use `UNSUPPORTED_ANDROID_IPI_CODE` for resize feature

Reply:
Yes @yoalex5, you are right on that, but the current InAppNative implementation does not allow us to send back an ErrorCode. As `PrebidNativeAdListener` consists of only 3 callbacks, namely:
* void onPrebidNativeLoaded(PrebidNativeAd ad);
* void onPrebidNativeNotFound();
* void onPrebidNativeNotValid();

`onPrebidNativeNotFound()` felt more suitable, thus used it to our rescue.